### PR TITLE
Update PPP for the Project

### PR DESCRIPTION
**Pull request by fungusta**

### CODE REVIEW

File: docs/team/fungusta.md [hunk 1]
```diff
@@ -19,16 +20,17 @@ Given below are my contributions to the project.
...
 * **Project management**:
-    * [to be added soon]
+    * Set up GitHub Issues and Milestones for tasks to be completed before certain deadlines.
+    * Contributed in reviewing Pull Requests made by other members of the team.
 
 * **Enhancements to existing features**:
     * Contact Finding by Address
-      * Finds and lists out all people in FABook that address contains any of the keywords input by user
+      * Finds and lists out all people in FABook that address contains any of the keywords input by user.
+      * Improved polymorphism of `Command` class by adding an abstract child `FindCommand` for find commands.
```
Comment (ReaganTan00) [inline docs/team/fungusta.md:29]: I think this can be a main point instead of sub-point

**Review by ReaganTan00 — APPROVED** [review]

File: docs/team/fungusta.md (hunk 1 above)
Comment (fungusta) [reply docs/team/fungusta.md:29]: I think because it was done when enhancing contact finding, should be ok? what you think?

**Review by shauneang — APPROVED** [review]

**Comment by shauneang** [comment]
I think you can merge once you are ready, lgtm

File: docs/team/fungusta.md (hunk 1 above)
Comment (ReaganTan00) [reply docs/team/fungusta.md:29]: Either way is fine 👍

**Pull request merged**